tests/unit/Core/Checkout/Customer/Validation/AddressValidationFactoryTest.php

```diff
@@ -21,7 +21,7 @@
  */
 #[Package('checkout')]
 #[CoversClass(AddressValidationFactory::class)]
-class AddressValidationFactoryTest extends TestCase
+class AddressValidationFactoryTest extends MonologTestCase
```

Comment:
Why don’t you use the PHPUnit\Framework\TestCase class?

Reply:
I don't know I didn't write it, it was already using the monlog base test class

Reply:
But yeah, look like its not using functionality from there - maybe it was an accidental import. I'll fix.

Reply:
Under the hood it uses the PHPUnit TestCase Class

https://github.com/Seldaek/monolog/blob/main/src/Monolog/Test/MonologTestCase.php